src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -26,6 +26,20 @@ const gridStyle = {
   paddingTop: '12px',
 };
 
+const useStyles = makeStyles({
+  minusMargin: {
+    left: 'calc(0px + 24px)',
+    width: 'calc(100% + 48px)',
+    position: 'relative',
+    marginLeft: '-48px',
+    '@media (min-width:768px)': {
+      left: 'calc(-117px + 154px)',
+      width: 'calc(100% + 154px)',
+      marginLeft: '-74px',
+    },
+  }
+});
```

Comment:
@haakemon You need to mark multiple lines before making a multiple line suggestion 😉 

```suggestion
const useStyles = makeStyles({
  minusMargin: {
    left: '24px',
    width: 'calc(100% + 48px)',
    position: 'relative',
    marginLeft: '-48px',
    '@media (min-width:768px)': {
      left: '37px',
      width: 'calc(100% + 154px)',
      marginLeft: '-74px',
    },
  }
});
```

Reply:
Learned something new today, I wasnt aware you could add a comment on multiple lines 😅 Thanks for the info!

Reply:
I saw someone else do it, and struggled for a while to figure out how it was done 😅

Reply:
lol, what was I thinking here 😂 Removed calcin latest commit, and also added correct space for screens between 768 and 992 😊